src/components/ServiceList.jsx

```diff
@@ -1,22 +1,61 @@
 import { Alert } from "@baltimorecounty/dotgov-components";
 import FilterList from "../components/FilterList";
-import { MostPopularServiceIconStyles } from "../styles";
-import React from "react";
+import React, { useState } from "react";
 import ServiceIconLink from "../components/ServiceIconLink";
 import useServices from "../hooks/useServices";
+import CheckBox from "./CheckBox";
+import Search from "./Search";
 
 const ServiceList = () => {
   const { hasError, serviceItems = [], isLoading } = useServices();
+  const [isChecked, setChecked] = useState(false);
```

Comment:
I'm assuming this is if popular services is checked?

Reply:
yes, you are right